### PR TITLE
 e2e/docker-dns-srv: test with TLS

### PR DESCRIPTION
**Pull request by gyuho**
To test https://github.com/coreos/etcd/pull/8651.

### CODE REVIEW

File: integration/fixtures/server-ca-csr-wildcard.json [hunk 1]
```diff
@@ -14,6 +14,9 @@
   ],
   "CN": "example.com",
   "hosts": [
-    "*.etcd.local"
+    "*.etcd.local",
+    "etcd.local",
```
Comment (gyuho) [inline integration/fixtures/server-ca-csr-wildcard.json]: NOTE: `"etcd.local",` can be removed with https://github.com/coreos/etcd/pull/8651. But keeping it here for older versions.

**Comment by xiang90** [comment]
lgtm. but this probably can be merged into our e2e test (make e2e test run under container evn in the future?)

**Comment by gyuho** [comment]
Sure, I will add those simple tests to semaphore.

**Comment by gyuho** [comment]
Later we can run e2e tests against this cluster.
c.f. https://github.com/coreos/etcd/issues/8502

**Pull request merged**